tests/coco_mock.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020–2021 Cirq on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Mocks backend calls for testing
+"""
+
+import json
+import requests
+from mockito import when, mock
+
+BASE_URL = 'https://meetiqm.com/api/'
```

Comment:
Please use a designated example domain instead of a real one (e.g. example.com)

Reply:
done